docs/packages.toml

```diff
@@ -77,6 +77,8 @@
 # Penopt.jl isn't a registered package yet
 # [Penopt]
 #     rev = "486f07d3f3a11f12012ea3ada702a3ee55c8fdc5"
+[Percival]
+    rev = "69842fcd05eaa80cff6cbe4435184ee066fd2826"
```

Comment:
This needs to go in the bottom section with JSO as the user: https://github.com/JuliaSmoothOptimizers/Percival.jl